ocaml/xapi/xapi_pusb.ml

```diff
@@ -31,6 +32,55 @@ let create ~__context ~uSB_group ~host ~other_config ~path
   debug "PUSB ref='%s' created" (Ref.string_of pusb);
   pusb
 
+let scan_start ~__context =
+  let open USB in
```

Comment:
Don't open `USB` - use it explicitly where you need it.

Reply:
As  create ~__context ~uSB_group:(Ref.null) ~host ~other_config:[] ~path:s.path ~vendor_id:s.vendor_id ....... will need to be  create ~__context ~uSB_group:(Ref.null) ~host ~other_config:[] ~path:s.USB.path ~vendor_id:s.USB.vendor_id .... . So USB will be use many times. Will just open more easier?

Reply:
updated